### DISCUSSION TITLE
Turning lanes

### DISCUSSION THREAD

**Discussion by KillerKingSolo** [organicmaps/organicmaps#9062] [Q&A]
There have been a lot of bug reports, a.k.a. suggestions to add turning lanes, but yet I haven't seen anything in development for over a year now in the bug reports, just wondering if everyone forgot about it or it's not going to happen.

**Comment by biodranik** [comment]
We have more than 2300 issues now. Nothing is forgotten. With enough donations and support all important issues can be handled sooner. Any help with the implementation is highly appreciated.
